buildspec.yml

```diff
@@ -18,7 +18,7 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - docker build -t $REPOSITORY_URI:$IMAGE_TAG .
+      - docker build -t $REPOSITORY_URI:$IMAGE_TAG --build-arg STAGE .
```

Comment:
In order for the argument to end up in the resulting image, the Dockerfile needs to be adapted aswell.

```
FROM python

ARG STAGE

ENV STAGE=$STAGE

...
```

Reply:
While on it, maybe change the base image from python:3.8 to python:3.8-slim